index.bs

```diff
@@ -196,6 +196,16 @@ spec: RFC4648; urlPrefix: https://tools.ietf.org/html/rfc4648
 }
 </pre>
 
+<pre class="link-defaults">
```

Comment:
I'd prefer us to avoid the link defaults if possible and just specify the exact link in each case. Previously we've had problems with people not realising the link was ambiguous and linking to the wrong thing because of a setting in defaults, so we removed it.